TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -99,6 +100,24 @@ private extension HomeViewController {
     }
 }
 
+// MARK: - Private
+private extension HomeViewController {
+    func profileButton(withImageData data: Data) {
```

Comment:
@AlexCornforth I made this into a separate function to allow the delegate method to be more slim. I took some time to try and make this work with `UIBarButtonItem` but i had no luck.